stimpacks/deploy/docker.go

```diff
@@ -32,9 +33,27 @@ func (d *Deploy) startDeployContainer(instance *Instance) {
 		d.log.Debug(scanner.Text())
 	}
 
-	envs := make([]string, len(instance.Spec.EnvironmentVars))
-	for i, e := range instance.Spec.EnvironmentVars {
-		envs[i] = fmt.Sprintf("%s=%s", e.Name, e.Value)
+	var envs []string
+	deprecatedHelmVersionSet := false
+	for _, e := range instance.Spec.EnvironmentVars {
+		if e.Name == "HELM_VERSION" {
+			d.log.Warn("The use of the HELM_VERSION environment variable for specifying Helm versions has been deprecated.  Use the `.spec.tools.helm` configuration for specifying the helm version to use.  See https://github.com/PremiereGlobal/stim/blob/master/docs/DEPLOY.md for more details.")
+			deprecatedHelmVersionSet = true
+		}
+		envs = append(envs, fmt.Sprintf("%s=%s", e.Name, e.Value))
+	}
+
+	if _, ok := instance.Spec.Tools["helm"]; ok {
+		if !deprecatedHelmVersionSet {
+			envs = append(envs, fmt.Sprintf("HELM_VERSION=%s", downloader.GetBaseVersion(instance.Spec.Tools["helm"].Version)))
+		} else {
+			d.log.Warn("Both `spec.tools.helm` and the deprecated HELM_VERSION environment variable are set.  HELM_VERSION is taking precedence")
```

Comment:
Might log the actual version we ended up using here too just so they know.

Reply:
fixed